clouddriver-aws/src/main/groovy/com/netflix/spinnaker/clouddriver/aws/deploy/ops/AbstractInstanceLoadBalancerRegistrationAtomicOperation.groovy

```diff
@@ -34,8 +31,7 @@ import groovy.transform.PackageScope
 import org.springframework.beans.factory.annotation.Autowired
 
 abstract class AbstractInstanceLoadBalancerRegistrationAtomicOperation implements AtomicOperation<Void> {
```

Comment:
Since you can't 'add to load balancer' in the ALB land, and you have to add to a 'target group', does it make sense to leave them the same?